client/src/unifycr.c

```diff
@@ -1931,7 +1931,10 @@ static int unifycr_init(int rank)
         if (env) {
             strcpy(external_data_dir, env);
         } else {
-            strcpy(external_data_dir, EXTERNAL_DATA_DIR);
+            fprintf(stderr, "UNIFYCR_EXTERNAL_DATA_DIR not set to an SSD path "
+                            "(i.e UNIFYCR_EXTERNAL_DATA_DIR=/mount/ssd):%s : %d\n",
+                            __FILE__, __LINE__);
```

Comment:
* This should probably use the DEBUG macro rather than print directly to the console.
* The path doesn't technically *have* to be on a SSD, so I might word this like "to an existing writable directory"
* I'd suggest /mnt instead of /mount for the example path to be more conventional

Reply:
@nedbass Okay, I had been using the DEBUG macro originally, but wasn't sure if it should show if DEBUG wasn't turned on. I will update.

Reply:
@nedbass There is a unifycr map to error numbers in unifycr.c of some kind it looks like? But they are only sometimes used. Also, I updated with the change to DEBUG and removed the reference to SSD.

Reply:
@dsikich thanks. I think error mapping and reporting would be a good area for us to revisit in a new  issue.